automated/linux/kvm-unit-tests/kvm-unit-tests.sh

```diff
@@ -82,6 +84,13 @@ else
   install
 fi
 
+if [ "${DEVICE}" = "JUNO-R2" ] || [ "${DEVICE}" = "juno-r2" ]; then
```

Comment:
I think I see why you do this, but why not just convert DEVICE to lower case so you don't have to have multiple case cases?

Reply:
@danrue only checks for lower case "juno-r2".